Find a functionally equivalent code:android.graphics.RadialGradient.copy:COMMENT
Method Modifier: protected   hidden      
Comment:/**
 * @hide
 */

@@ -5,11 +5,11 @@
             copy = new RadialGradient(mX, mY, mRadius, mColors.clone(), mPositions != null ? mPositions.clone() : null, mTileMode);
             break;
         case TYPE_COLOR_CENTER_AND_COLOR_EDGE:
-            copy = new RadialGradient(mX, mY, mRadius, mColor0, mColor1, mTileMode);
+            copy = new RadialGradient(mX, mY, mRadius, mCenterColor, mEdgeColor, mTileMode);
             break;
         default:
             throw new IllegalArgumentException("RadialGradient should be created with either " + "colors and positions or center color and edge color");

